### PR TITLE
update xcavate logo

### PR DESCRIPTION
**Pull request by ganesh1997oli**

### CODE REVIEW

**Review by TarikGul — CHANGES_REQUESTED** [review]
2 fixes required. Linter, and decreasing the image size

**Comment by ganesh1997oli** [comment]
> 2 fixes required. Linter, and decreasing the image size

all checks have passed

**Comment by ganesh1997oli** [comment]
> > 2 fixes required. Linter, and decreasing the image size
> 
> all checks have passed

hey @TarikGul would you mind merging, please!

**Comment by ganesh1997oli** [comment]
> > > 2 fixes required. Linter, and decreasing the image size
> > 
> > 
> > all checks have passed
> 
> hey @TarikGul

hey @TarikGul, does it take long time to merge this PR?

**Review by TarikGul — CHANGES_REQUESTED** [review]
I think the last logo needs to be removed completely no?

**Comment by ganesh1997oli** [comment]
> I think the last logo needs to be removed completely no?

yes completely!

**Comment by ganesh1997oli** [comment]
Suddenly it PR shows conflict and after fixing I get test fails, can you tell me which test fail?

File: packages/apps-config/src/endpoints/testingRelayPaseo.ts [hunk 1]
```diff
@@ -249,7 +249,20 @@ export const testParasPaseo: Omit<EndpointOption, 'teleport'>[] = [
     text: 'Xcavate',
     ui: {
       color: '#FF0083',
-      logo: chainsXcavatePNG
+      logo: chainsNewxcavPNG
+    }
+  },
+  {
+    homepage: 'zeitgeist.pm',
```
Comment (TarikGul) [inline packages/apps-config/src/endpoints/testingRelayPaseo.ts:256]: It's saying this is a duplicate, when you were fixing conflicts it looks like you fixed them wrong.

**Review by TarikGul — COMMENTED** [review]
Also as mentioned you need to remove the old PNG - `chainsXcavatePNG`. It's not being used anymore so it shouldn't be in the repo.

**Pull request closed without merging**